### PR TITLE
[docs]: Move files from main repo

### PR DESCRIPTION
**Pull request by outoftardis**
Signed-off-by: Ekaterina Mekhnetsova <mekkatya@gmail.com>

- move some docs from iroha repository (see pr: https://github.com/hyperledger/iroha/pull/2879)
- update the chapter on genesis (partially addresses #171)

### CODE REVIEW

File: src/api/index.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # API
```
Comment (appetrosyan) [inline src/api/index.md:1]: Keep this in the repo

File: src/guide/advanced/hot-reload.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# How to hot reload Iroha in a Docker container
```
Comment (appetrosyan) [inline src/guide/advanced/hot-reload.md:1]: Keep this in the repo.

**Review by appetrosyan — CHANGES_REQUESTED** [review]

File: src/guide/advanced/hot-reload.md (hunk 2 above)
Comment (outoftardis) [reply src/guide/advanced/hot-reload.md:1]: kept in the repo, check: https://github.com/hyperledger/iroha/pull/2879

**Review by appetrosyan — APPROVED** [review]

File: src/guide/configure/genesis.md [hunk 3]
```diff
@@ -1,19 +1,29 @@
...
 <<< @/guide/configure/configs/genesis.json
 
 :::
 
+The **genesis account** is specified in the
+[peer configuration](./peer-configuration.md#genesis) file,
+`configs/peer/config.json`. This is the account that will submit the
+genesis block. The genesis account is like a super user account that has
+elevated privileges, but only during the genesis round. It should be signed
+by one of the peers (has the public key of this peer).
```
Comment (WRRicht3r) [inline src/guide/configure/genesis.md:17]: What has the key? the Genesis block? Please make this more clear

File: src/guide/glossary.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+as run-time configuration updates.
+
+### Kura (Warehouse)
+
+Persistence-related logic. It handles storing the blocks, log rotation,
+block storage folder rotation, etc.
+
+### Kagami(Teacher and Exemplar and/or looking glass)
+
+Generator for commonly used data. Can generate cryptographic key pairs,
```
Comment (WRRicht3r) [inline src/guide/glossary.md:60]:
```suggestion
Generator for commonly used data. It can generate cryptographic key pairs,
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

File: src/guide/configure/genesis.md (hunk 3 above)
Comment (outoftardis) [reply src/guide/configure/genesis.md]: rephrased

**Pull request merged**